scripts/models/Command.js

```diff
@@ -83,7 +83,13 @@ app.Models.CommandsCollection = Backbone.AuthCollection.extend({
                     that.add(new app.Models.Command(incomingCommand, { device: that.device }));
                 });
             },
-            complete: function () {
+            complete: function (xhr) {
+                // prevent polling if error occur
+                if (xhr.status >= 400) {
+                    app.vent.trigger("notification", app.Enums.NotificationType.Error, xhr);
```

Comment:
nope. notification here means application-wide notification system. In other words it's just a single method to notify user about the issue.